skipper-mixin/README.md

```diff
@@ -0,0 +1,23 @@
+# Skipper Mixin
+
+The Skipper Mixin is a set of configurable, reusable, and extensible alerts and
+dashboards based on the metrics exported by the MySQLd Exporter. The mixin creates
+recording and alerting rules for Prometheus and suitable dashboard descriptions
+for Grafana.
+
+To use them, you need to have `mixtool` and `jsonnetfmt` installed. If you
+have a working Go development environment, it's easiest to run the following:
+```bash
+$ go get github.com/monitoring-mixins/mixtool/cmd/mixtool
+$ go get github.com/google/go-jsonnet/cmd/jsonnetfmt
+```
+
+You can then build the Prometheus rules files `alerts.yaml` and
+`rules.yaml` and a directory `dashboard_out` with the JSON dashboard files
+for Grafana:
+```bash
+$ make build
+```
+
+For more advanced uses of mixins, see
```

Comment:
It would be useful to have the most basic example on what to do with the build output before the 'For more advanced uses...' link.